### PR TITLE
Show an error message when session storage is disabled

### PR DESCRIPTION
**Pull request by thivi**
Resolves https://github.com/wso2/product-is/issues/10435

An error message is shown when `sessionStorage` is disabled in the browser settings. 

image

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/admin-portal.ts [hunk 1]
```diff
@@ -3004,6 +3004,13 @@ export const adminPortal: AdminPortalNS = {
             },
             title: "You are not authorized"
         },
+        sessionStorageDisabled: {
+            subtitles: {
+                0: "It looks like your browser's settings don't allow data to be stored.",
+                1: "Please enable storing data on your browser for this app to function properly."
```
Comment (DimalChandrasiri) [inline modules/i18n/src/translations/en-US/portals/admin-portal.ts:3010]: IMO this message is vague and the user might get confused. @jeradrutnam any thoughts regarding this?

**Review by DimalChandrasiri — APPROVED** [review]

File: modules/i18n/src/translations/en-US/portals/admin-portal.ts (hunk 1 above)
Comment (jeradrutnam) [reply modules/i18n/src/translations/en-US/portals/admin-portal.ts:3010]: Yes, same thought I had. Better to rephrase it.

Comment (jeradrutnam) [reply modules/i18n/src/translations/en-US/portals/admin-portal.ts:3010]: May we can say like "**Cookies are disabled in your browser.** To use this application, you have to enable cookies in your Web browser settings. For more information on How to enable cookies, see Help section on your Web browser"

Comment (thivi) [reply modules/i18n/src/translations/en-US/portals/admin-portal.ts]: Updated by 058471f

File: modules/i18n/src/translations/si-LK/portals/user-portal.ts [hunk 2]
```diff
@@ -1309,6 +1309,13 @@ export const userPortal: UserPortalNS = {
                 1: "කරුණාකර වෙනත් ගිණුමක් සමඟ පුරනය වන්න."
             },
             title: "ඔබට අවසර නැත"
+        },
+        sessionStorageDisabled: {
+            subtitles: {
+                0: "මෙම යෙදුම භාවිතා කිරීමට, ඔබේ වෙබ් බ්‍රව්සර සැකසුම් තුළ කුකී සක්‍රීය කළ යුතුය.",
```
Comment (brionmario) [inline modules/i18n/src/translations/si-LK/portals/user-portal.ts:1315]:
```suggestion
                0: "මෙම ඇප් එක භාවිතා කිරීමට, ඔබේ වෙබ් බ්‍රව්සර සැකසුම් තුළ කුකී සක්‍රීය කළ යුතුය.",
```

**Review by brionmario — APPROVED** [review]

**Pull request merged**